terraform/cloud-platform-components/README.md

```diff
@@ -42,10 +66,41 @@ resource "aws_iam_role_policy" "app" {
 
 This can easily be configured as part of a user environment's resources, along with the required namespace annotation (see the [kiam docs](https://github.com/uswitch/kiam#overview)).
 
-## Prometheus to Slack Alerting Routes
+## Kuberos
+To allow users to authenticate we spin up a `Kuberos` pod. Kuberos is an OIDC authentication helper for Kubernetes' kubectl and enables users to perform queries against the clusters API. 
 
-#### 1. Create a Slack incoming webhook
+Again, we use the stable Helm chart for the deployment.
+
+## Metrics-server
+Metrics-server allows us to perform resource queries against the cluster. Commands like `kubectl top pods` allow us to diagnose resource constraints. 
+
+## Nginx-ingress
+A vital component in the cluster. The Nginx-ingress controller is a daemon, deployed as a Kubernetes Pod, that watches the apiserver's /ingresses endpoint for updates to the Ingress resource. Its job is to satisfy requests for Ingresses.
+
+## Prometheus
+We utilise [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) to deploy Prometheus onto the Cloud Platform. Once installed a `DaemonSet` of exporters is deployed scraping metrics from across the cluster. Grafana and AlertManager are also deployed as part of this chart along with relevant proxies. 
+
+### Persistent Volumes with Prometheus
+
+To maintain data across deployments and version upgrades data must be persisted to a volume (AWS EBS) other than emptyDir, allowing it to be reused by pods after upgrade. Please see the following documentation by CoreOS on how to do this. https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/storage.md
+
+This has previously been achieved by applying an individual storage class manifest and referencing it in the values.yaml Prometheus-operator Helm chart.
 
+### Adding Pingdom Alerts to monitor Prometheus and Alermanager Externally
+Prometheus and AlertManager will be behind an OIDC proxy with GitHub credentials required to view the GUI. However, the /-/healthy endpoint for each application will be exposed directly to the internet.
+
+```
+https://$PROMETHEUS_URL$/-/healthy
+https://$ALERTMANAGER_URL$/-/healthy
+```
+
+To expose the /-/healthy endpoint, an additional path entry is required in the Ingress object. Please see (oidc-proxy.yaml)[monitoring/oidc-proxy.yaml] for details.
+
+A pingdom alert should be setup (with appropriate alert recipients) to the /healthy endpoints for each application described above.
+
+### Prometheus to Slack Alerting Routes
+
```

Comment:
point to Sablu's README to avoid duplication?

Reply:
As discussed. This is deliberate.